src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapGood.java

```diff
@@ -1,12 +1,14 @@
 package com.google.checkstyle.test.chapter3filestructure.rule332nolinewrap; //ok
 
+import static java.math.BigInteger.ZERO;
```

Comment:
I shifted this after all the other import statements and i have this job failing
https://travis-ci.org/checkstyle/checkstyle/jobs/212226897

This task fails and the reason mentioned is 
```bash
[WARN] /home/travis/build/checkstyle/checkstyle/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapGood.java:8: Import statement for 'java.math.BigInteger.ZERO' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line. [CustomImportOrder]
```

How should I proceed?